powermon/powermon-qpigs.yaml

```diff
@@ -7,18 +7,25 @@ device:
   port:
     type: test
     protocol: PI30MAX
-  commands:
-  - command: QPIGS
-    outputs:
-    - name: screen
-      # format: hass
-      # format: raw
-      # format: simple
-      # format: table
-      format: htmltable
-      #keep_case: True
-      #remove_spaces: False
-      #extra_info: True
-      tag: Test_Inverter
-      filter: ^ac|^pv
-loop: once
\ No newline at end of file
+
+scheduling:
+  loop: once
+  schedules:
+  - name: QPIGS to screen
+    type: once
+    commands:
+    - command: QPIGS
+      type: basic #so far not neccessary, might remove
```

Comment:
what did you envision as other options? (suspect that if you think there complicated use cases then you are probably correct - every time I have decided to assume only the simple case I have been bitten)

Reply:
Just seemed like a spot that would need some flexibility. Sounds like you need a different command type for devices that push data. A different command type will help differentiate the setup.